src/BinaryBuilder.jl

```diff
@@ -10,4 +10,24 @@ include("Auditor.jl")
 include("DockerRunner.jl")
 include("Dependency.jl")
 
-end # module
\ No newline at end of file
+export shortname
+
+"""
+    shortname(p::Platform)
+
+Get a convenient symbol representation of the given platform.
+
+# Examples
+```jldoctest
+julia> shortname(Linux(:i686))
+:linux32
+
+julia> shortname(MacOS())
+:osx64
+```
+"""
+shortname(p::Linux) = Symbol("linux", wordsize(p))
```

Comment:
I don’t think this will work for aarch64 or ppc64le, right? We need to encode the arch or something.

Reply:
Oh yes you're right